src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  object-fit: contain;
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+
+}
+
+.card__name {
+  font-weight: 500;
+  line-height: 18px;
+  align-self: center;
+  margin-top: 40px;
+  margin-bottom: 0;
+  color: #060b35;
+  font-size: 12px;
+  width: 166px;
+}
+
+.card__code {
+  font-weight: 400;
+  margin-top: 4px;
+  color: #616070;
+  font-size: 10px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.card__reviews {
+  align-self: end;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.price__text {
+  display: flex;
+  align-items: end;
+  font-size: 12px;
+  color: #616070;
+}
+
+.price__number {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  text-decoration: none;
+  background-color: #00acdc;
+  color: white;
+  font-size: 14px;
+  border-radius: 5px;
+  margin: 16px 0;
+  height: 40px;
+  text-transform: uppercase;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: white;
+  border: 1px solid #00acdc;
```

Comment:
You shouldn't set new border no `hover` state, you should change only border color